packages/dhis2/src/Adaptor.js

```diff
@@ -442,10 +492,17 @@ export function update(
  * });
  * @example <caption>all programs for an organization unit</caption>
  * get('programs', { orgUnit: 'TSyzvBiovKh', fields: '*' });
- * @example <caption>a single tracked entity instance by a unique external ID</caption>
- * get('trackedEntityInstances', {
- *   ou: 'DiszpKrYNg8',
- *   filter: ['flGbXLXCrEo:Eq:124', 'w75KJ2mc4zz:Eq:John'],
+ * @example <caption>a single tracked entity instance by a unique entity ID. Either `program`, `trackedEntityType` or `trackedEntities` should be specified in the params. See {@link https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#tracked-entities-get-apitrackertrackedentities TrackedEntities docs}</caption>
```

Comment:
I think these examples look a bit too complicated and we should do more explanation in the main docs body. Let me think about that